src/main/java/net/pms/network/mediaserver/javahttpserver/RequestHandler.java

```diff
@@ -1369,11 +1370,15 @@ private static boolean filterIp(InetAddress inetAddress) {
 
 	private static void logMessageSent(HttpExchange exchange, String response, InputStream iStream, RendererConfiguration renderer) {
 		StringBuilder header = new StringBuilder();
+		boolean isXml = false;
 		for (Map.Entry<String, List<String>> headers : exchange.getResponseHeaders().entrySet()) {
 			String name = headers.getKey();
 			if (StringUtils.isNotBlank(name)) {
 				for (String value : headers.getValue()) {
 					header.append("  ").append(name).append(": ").append(value).append("\n");
+					if ("content-type".equalsIgnoreCase(name) && "/XML".contains(value.toUpperCase())) {
```

Comment:
I think `"/XML".contains(value.toUpperCase())` is backwards. It should be `value.toUpperCase().contains("/XML")`, no?

Reply:
Good point!